packages/openapi3/src/openapi.ts

```diff
@@ -118,6 +123,37 @@ export async function $onEmit(context: EmitContext<OpenAPI3EmitterOptions>) {
   await emitter.emitOpenAPI();
 }
 
+type IrrelevantOpenAPI3EmitterOptionsForObject = "file-type" | "output-file" | "new-line";
+
+/**
+ * Get the OpenAPI 3 document records from the given program. The documents are
+ * returned as a JS object.
+ *
+ * @param program The program to emit to OpenAPI 3
+ * @param options OpenAPI 3 emit options
+ * @returns An array of OpenAPI 3 document records.
+ */
+export async function getOpenAPI3(
+  program: Program,
+  options: Omit<OpenAPI3EmitterOptions, IrrelevantOpenAPI3EmitterOptionsForObject> = {}
+) {
```

Comment:
```suggestion
): Promise<OpenAPI3ServiceRecord[]> {
```

think its good to be explicit for an exported function